scripts/merged-prs/distribute.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
```

Comment:
There are minor differences between this script and `scripts/distribute-merged-prs.sh`. Why not just use one?